docs/src/modules/components/MarkdownElement.js

```diff
@@ -431,7 +431,9 @@ const Root = styled('div')(
       // Avoid layout jump
       display: 'inline-block',
       // Avoid very sharp edges
-      borderRadius: 2,
+      borderRadius: 12,
+      border: '1px solid',
+      borderColor: `var(--muidocs-palette-divider, ${lightTheme.palette.divider})`,
```

Comment:
This feels very/too ambitious.

examples: https://deploy-preview-40848--material-ui.netlify.app/material-ui/getting-started/support/#bug-reproductions

or 

<img width="304" alt="SCR-20240221-qkcp" src="https://github.com/mui/material-ui/assets/3165635/4a875e3e-ffcd-4792-87ea-4b1f107f06b4">

```suggestion
      borderRadius: 2,
```

Reply:
I know, yeah. It might be too early to add it in this PR but it's something I want to do, as we've been adding borders inconsistently in the images themselves rather than just solve this on the code 😬